lib/engines/http.ex

```diff
@@ -27,6 +37,13 @@ defmodule Engines.Http do
 
   ## Private
 
+  defp sign_url(url, date \\ Date.now) do
+    headers = HashDict.new
+    AWSAuth.sign_url(amazon_access_key_id, amazon_secret_access_key,
+                     "GET", url, aws_s3_region, "s3", headers,
+                     %{date | ms: 0})
```

Comment:
The milliseconds are truncated here because of the problem described in bryanjos/aws_auth#8. It ccan be removed if a new version of aws_auth gets released.